src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +125,73 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              {View: WindowsHelloErrorMessageView},
+              {selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c">\
+              {{i18n code="enroll.windowsHello.subtitle" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: null}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c">\
+              {{i18n code="enroll.windowsHello.subtitle.loading" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'loading'}
+          }));
+        }
+
+        result.push(FormType.View({View: '<div class="okta-waiting-spinner hide"></div>'}));
```

Comment:
@lilyhsiao-okta it's less work to switch to to existing spinner - all the logic is already there.

@alexeystadnikov-okta a reference example:
https://github.com/okta/okta-signin-widget/blob/b28719fc373a9cd0252ddb7d915bd7e88677d0ac/src/models/PrimaryAuth.js#L81